torch_xla/csrc/init_python_bindings.cpp

```diff
@@ -516,24 +515,6 @@ py::object GetRevisions() {
   return py_dict;
 }
 
-std::vector<py::bytes> Rendezvous(int ordinal, const std::string& tag,
```

Comment:
FYI @amithrm we are removing the mesh server based Rendezvous as part of the XRT clean up